bin/jest.js

```diff
@@ -64,8 +64,14 @@ function _findChangedFiles(dirPath) {
   return deferred.promise;
 }
 
-function _onResultReady(config, result) {
-  return defaultTestResultHandler(config, result);
+function _onResultReady(argv, config, result) {
+  if (argv.detailedResults) {
+    return defaultTestResultHandler
+    .printDetailedTestResult(config, result);
```

Comment:
Nit: please indent this lint. There are a bunch of other places where you don't do this when wrapping but that's really tough to read.